docs/content/docs/build/packages.md

```diff
@@ -1,14 +1,72 @@
 ---
 title: Packages
 menuPosition: 7
-editor: tylerbutler
-status: unwritten
+author: tylerbutler
 ---
 
-Explain what the fluid-framework package is.
-Explain the namespaces (scopes).
-List of all the packages in fluid-framework.
+The Fluid Framework is a multi-layered system consisting of dozens of individual npm packages. Most developers will only
+need two or three of these packages for typical Fluid development: The `fluid-framework` package, which contains the
+public API for the Fluid Framework, and a service-specific client package, such as the
+`@fluidframework/tinylicious-client` package.
 
-Experimental scope and packages.
+## Primary API: fluid-framework
 
-List other scopes and their meaning (or link to the wiki).
+The `fluid-framework` package bundles a collection of Fluid Framework client packages for easy use when paired with a
+corresponding service client package, such as the `@fluidframework/azure-client` package. The `fluid-framework` package
+consists primarily of two portions: the `IFluidContainer` interface and a selection of distributed data structures
+(DDSes).
+
+### IFluidContainer
```

Comment:
I'm concerned about the implications of talking about the interface vs the object. The main problem I have is that talking about interface means talk about Typescript. This is not inclusive to the JavaScript community and honestly confusing IMO. I know Matt updated the readme to support this syntax but I feel that talking about interfaces are to OO of a concept for JS devs.

Reply:
I would rather this section talk about `FluidContainer`

Reply:
Where do I see the FluidContainer type show up? I.e. is there any time where I am going to get back a FluidContainer as opposed to an IFluidContainer?